spec/dummy_winrm_service.rb

```diff
@@ -0,0 +1,24 @@
+module Dummy
```

Comment:
Are you doing this because of some lulz when combining instance doubles with instance_variable_get?